homeassistant/components/hvv_departures/config_flow.py

```diff
@@ -122,31 +125,32 @@ async def async_step_station_select(self, user_input=None):
 
     @staticmethod
     @callback
-    def async_get_options_flow(config_entry):
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> OptionsFlowHandler:
         """Get options flow."""
         return OptionsFlowHandler(config_entry)
 
 
 class OptionsFlowHandler(config_entries.OptionsFlow):
     """Options flow handler."""
 
-    def __init__(self, config_entry):
+    def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
         """Initialize HVV Departures options flow."""
         self.config_entry = config_entry
         self.options = dict(config_entry.options)
-        self.departure_filters = {}
-        self.hub = None
+        self.departure_filters: dict[str, Any] = {}
 
     async def async_step_init(self, user_input=None):
         """Manage the options."""
         errors = {}
         if not self.departure_filters:
 
             departure_list = {}
-            self.hub = self.hass.data[DOMAIN][self.config_entry.entry_id]
+            hub: GTIHub = self.hass.data[DOMAIN][self.config_entry.entry_id]
```

Comment:
I had to add typing to `self.departure_filters` and `self.hub` to make it pass, and I felt it was better to type the hub only in `async_step_init`.